src/graphql/utilities/ast_from_value.py

```diff
@@ -117,7 +117,10 @@ def ast_from_value(value: Any, type_: GraphQLInputType) -> Optional[ValueNode]:
         if isinstance(serialized, int):
             return IntValueNode(value=f"{serialized:d}")
```

Comment:
I think we should then set `value=str(serialized)` here as well, just for symmetry with the case below.

Reply:
Makes sense, I've now made that change.